src/main/java/pl/javastart/task/Season.java

```diff
@@ -1,5 +1,43 @@
 package pl.javastart.task;
 
+import java.util.Arrays;
+import java.util.Scanner;
+
 public enum Season {
-    //uzupełnij
+    SPRING(new String[]{"marzec", "kwiecień", "maj"}, "wiosna"),
+    SUMMER(new String[]{"czerwiec", "lipiec", "sierpień"}, "lato"),
+    AUTUMN(new String[]{"wrzesień", "październik", "listopad"}, "jesień"),
+    WINTER(new String[]{"grudzień", "styczeń", "luty"}, "zima");
+
+    Season(String[] months, String plTranslation) {
+        this.months = months;
+        this.plTranslation = plTranslation;
+    }
+
+    private final String[] months;
+    private final String plTranslation;
+
+    public String[] getMonths() {
+        return months;
+    }
+
+    public String getPlTranslation() {
+        return plTranslation;
+    }
+
+    public static void findSeason(Scanner scanner) {
+        String choice = scanner.nextLine();
+        Season season = null;
+        for (Season value : Season.values()) {
+            if (choice.toLowerCase().equals(value.getPlTranslation())) {
+                season = value;
+            }
+        }
+        if (season != null) {
+            System.out.println("W tej porze roku są następujące miesiące:");
+            System.out.println(Arrays.toString(season.getMonths()));
+        } else {
+            System.out.println("Wprowadzono nieprawidłowe dane.");
+        }
+    }
```

Comment:
W tej klasie nie powinieneś używać Scannera. Idea jest taka, żeby w aplikacji rozdzielać klasy z logiką, od klas, które służą do interakcji z użytkownikiem.
Stwórz raczej metodę, która przyjmuje String i przekształci to na wartość Season, a następnie wywołaj ją w metodzie run() np.:
```
String userInput = scanner...
Season season = Season.from(userInput);
...wyświetlenie season
```